dist/images/ovnkube.sh

```diff
@@ -117,7 +134,7 @@ ovn_db_host=""
 # =========================================
 
 setup_ovs_permissions () {
-  if [ ${ovs_user_id:-XX} != "XX" ]; then
+  if [[ ${ovs_user_id:-XX} != "XX" ]] ; then
```

Comment:
if you're going to do style fixes on this:

    if [[ -n "${ovs_user_id}" ]]; then

(and again below)